packages/atomic/create-atomic/scripts/packageTemplate.json

```diff
@@ -12,6 +12,6 @@
     "site:init": "netlify init",
     "site:link": "netlify link",
     "site:deploy": "npm run site:link && npm run build && netlify env:import .env && netlify deploy --prod --open",
-    "postinstall": "npm run setup-lambda && npm run setup-cleanup"
+    "postinstall": "npm run setup-lambda && npm run setup-cleanup && prettier --write . --loglevel warn"
```

Comment:
Just curious, what motivate this change 🤔 
Also, are you sure prettier will be there?

Reply:
Yes prettier is installed!

Especially when downloading the markup from the SIS, it's a way better experience to have everything formatted properly!

Reply:
Oh yeah true, ["prettier": "2.6.0",](https://github.com/coveo/cli/blob/d48e2c9d09430a8feb6baff4ad8c867ef0d97969/packages/atomic/template/package.json#L20) ma bad.